### ISSUE TITLE
Manage_Settings_DeveloperTool: The Clear search input button is wrapped in a <ul> which is given a role toolbar.

### ISSUE THREAD

**Issue by nasc17** [microsoft/azuredatastudio#17968]
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
Environment Details:
Version: 1.35.0-insider (user setup)
Commit: 2bf3c28370dd6dd706771c83d7820da6db87eb1d
Date: 2021-12-20T06:30:03.618Z
VS Code: 1.58.0
Electron: 12.0.9
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.22000

**Repro Steps:**
Launch the application-Azure Data Studio
Tab till settings button and press enter.
Tab till search field.
Tab till clear search input button and inspect using Developer tool.
**Actual:**
The Clear search input button is wrapped in a `<ul>` which is given a role toolbar.

![image](https://user-images.githubusercontent.com/69922333/146830203-a8a5ddc5-e269-4fc8-846c-9825b99310ab.png)

**Expected:**
The clear search input button should be given a role as presentation.
**User Impact:**
Screenreader users will get confused if the role is not correctly defined.

MAS Reference:
MAS 4.1.2 – Name, Role, Value

**Comment by alanrenmsft** [comment]
@nasc17 could you please check the latest VSCode and see whether it is behaving the same? if so, please open an issue in VSCode.

**Comment by nasc17** [comment]
@alanrenmsft  I will close this issue and move to VSCode
<img width="904" alt="image" src="https://user-images.githubusercontent.com/69922333/146980501-27a42321-99da-4b39-b1a6-f6dc17ffdc93.png">
